tests/MimeTest.php

```diff
@@ -49,6 +49,13 @@ public function testWebp()
         $this->assertFileHasMimeType('sample.webp', 'image/webp');
     }
 
+    public function testDetermineDimensions()
+    {
+        $file = new \SplFileObject(__DIR__ . '/files/sample.png', 'r');
+
+        $this->assertSame([16, 16], $this->detector->determineDimensions($file));
```

Comment:
Can you add a new test at a different dimension? Just a small, single color image would work. Ideally an image that isn't square, like 3x7?

Reply:
Ok. I will do that.

Reply:
This test has been added via commit 558fe1.